interface-definitions/include/openconnect-config-per-x.xml.i

```diff
@@ -0,0 +1,54 @@
+<!-- include start from openconnect-config-per-x.xml.i -->
+<node name="config-per-x">
```

Comment:
I get your idea but I believe we can find a better name for this node than `config-per-x`. Like `external-config` maybe.